src/BloomBrowserUI/bookEdit/js/readerToolsModel.ts

```diff
@@ -1063,6 +1063,21 @@ class ReaderToolsModel {
       returnVal.push(new DataWord(words[i]));
     }
 
+    // inform the user if the list was truncated
+    var accordion: Window = iframeChannel.getAccordionWindow();
+    var msgDiv: JQuery = $(accordion).find('#allowed-word-list-truncated');
+
+    // We are reducing the max by 10 here in case there were a couple of duplicates that were removed after the list
+    // was truncated, causing the total number of words to now be slightly less than the max.
+    if (words.length < model.maxAllowedWords - 10) {
```

Comment:
The server removes duplicates found in a single file. That still leaves the possibility that duplicates exist between files.